src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ERR.cs

```diff
@@ -21,14 +21,16 @@ internal static partial class Crypto
         internal static extern IntPtr ErrReasonErrorString(ulong error);
 
         [DllImport(Libraries.CryptoNative, CharSet = CharSet.Ansi)]
```

Comment:
Since the marshaler isn't doing string interpretation do we want to remove the CharSet attributes?